test/request-response-samples/update__complex_extension_with_array_attributes/request.json

```diff
@@ -8,14 +8,14 @@
         "xmlns": "urn:relationships_2013_2.lists.webservices.netsuite.com"
       }
     },
+    "firstName": "Allister",
```

Comment:
Why did you prefer modifying an existing test in favour of creating a new one? Was the "old" test wrong in any sense?
I'm just asking, because "normally" rewriting tests in order to get a code running is very bad practice - unless the test was obviously wrong or not testing things at all. :smiley: